src/service_cmd/runner/runner.go

```diff
@@ -9,18 +9,27 @@ import (
 	pb "github.com/lyft/ratelimit/proto/envoy/service/ratelimit/v2"
 	pb_legacy "github.com/lyft/ratelimit/proto/ratelimit"
 
+	"fmt"
 	"github.com/lyft/ratelimit/src/config"
 	"github.com/lyft/ratelimit/src/redis"
 	"github.com/lyft/ratelimit/src/server"
 	"github.com/lyft/ratelimit/src/service"
 	"github.com/lyft/ratelimit/src/settings"
+	logger "github.com/sirupsen/logrus"
 )
 
 func Run() {
-	srv := server.NewServer("ratelimit", settings.GrpcUnaryInterceptor(nil))
-
 	s := settings.NewSettings()
 
+	logLevel, err := logger.ParseLevel(s.LogLevel)
+	if err != nil {
+		fmt.Printf("Could not parse log level. %v\n", err)
```

Comment:
we probably want to exit if the intended log level can't be set. i.e `logger.Fatalf(<the message you have here>)`